DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_bum_traffic_bridge_with_rif.py

```diff
@@ -0,0 +1,430 @@
+            {"device": bridge, "type": "bridge"}]}])
+    assert out[0][device_host_name]["rc"] == 0, f"Verify that bridge created.\n{out}"
+
+    out = await IpLink.set(
+        input_data=[{device_host_name: [
+            {"device": bridge, "operstate": "up"}]}])
+    assert out[0][device_host_name]["rc"] == 0, f"Verify that bridge set to 'UP' state.\n{out}"
+
+    out = await IpLink.set(
+        input_data=[{device_host_name: [
+            {"device": port, "master": bridge, "operstate": "up"} for port in ports]}])
+    err_msg = f"Verify that bridge entities set to 'UP' state and links enslaved to bridge.\n{out}"
+    assert out[0][device_host_name]["rc"] == 0, err_msg
+
+    out = await IpAddress.add(
+        input_data=[{device_host_name: [
+            {"dev": bridge, "prefix": "100.1.1.253/24"}]}])
+    assert out[0][device_host_name]["rc"] == 0, f"Failed to add IP address to bridge.\n{out}"
+
+    out = await IpLink.show(input_data=[{device_host_name: [{"device": ports[0], "cmd_options": "-j"}]}],
+                            parse_output=True)
+    assert out[0][device_host_name]["rc"] == 0, f"Failed to display device attributes.\n{out}"
+
+    dev_attributes = out[0][device_host_name]["parsed_output"]
+    self_mac = dev_attributes[0]["address"]
+    srcMac = "00:00:AA:00:00:01"
+
+    address_map = (
+        # swp port, tg port,    tg ip,     gw,        plen
+        (ports[0], tg_ports[0], "1.1.1.2", "1.1.1.1", 24),
+        (ports[1], tg_ports[1], "1.1.1.3", "1.1.1.1", 24)
+    )
+
+    dev_groups = tgen_utils_dev_groups_from_config(
+        {"ixp": port, "ip": ip, "gw": gw, "plen": plen}
+        for _, port, ip, gw, plen in address_map
+    )
+
+    await tgen_utils_traffic_generator_connect(tgen_dev, tg_ports, ports, dev_groups)
+
+    streams = {
+        "Bridged_UnknownL2UC": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcMac": srcMac,
+            "dstMac": "00:00:BB:11:22:33",
+            "frameSize": 96,
+            "protocol": "0x6666",
+            "type" :"raw",
+        },
+        # # TODO: FIX
+        # "STP_BPDU": {
+        #     "ip_source": dev_groups[tg_ports[0]][0]["name"],
+        #     "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+        #     "root_id": "8000.34:ef:b6:ec:26:c3",
+        #     #"protocolVId": "00",
+        #     "srcMac": srcMac,
+        #     "dstMac": "01:80:C2:00:00:00",
+        #     "frameSize": 96,
+        #     "protocol": "ip",
+        #     "type" :"raw"
+        # },
+        "BridgedLLDP": {
+            "ip_source": dev_groups[tg_ports[0]][0]["name"],
+            "ip_destination": dev_groups[tg_ports[1]][0]["name"],
+            "srcMac": srcMac,
+            "dstMac": "01:80:c2:00:00:0e",
+            "frameSize": 96,
+            "protocol": "0x88cc",
+            "type" :"raw"
+        },
+        #  # TODO: FIX
```

Comment:
Is this the issue with LLC frames? Do we have a bug for it? Why we can send LACP and cannot send this one?